tests/templates/kuttl/kerberos/30-access-kafka.txt.j2

```diff
@@ -0,0 +1,116 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: access-kafka
+spec:
+  template:
+    spec:
+      serviceAccountName: test-sa
+      containers:
+        - name: access-kafka
+          image: docker.stackable.tech/stackable/kafka:{{ test_scenario['values']['kafka'] }}-stackable0.0.0-dev
+          command:
+            - /bin/bash
+            - /tmp/script/script.sh
+          env:
+            - name: KRB5_CONFIG
+              value: /stackable/kerberos/krb5.conf
+            - name: KAFKA_OPTS
+              value: -Djava.security.krb5.conf=/stackable/kerberos/krb5.conf
+          volumeMounts:
+            - name: script
+              mountPath: /tmp/script
+            - mountPath: /stackable/tls-ca-cert-mount
+              name: tls-ca-cert-mount
+            - name: kerberos
+              mountPath: /stackable/kerberos
+      volumes:
+        - name: script
+          configMap:
+            name: access-kafka-script
+        - name: kerberos
+          ephemeral:
+            volumeClaimTemplate:
+              metadata:
+                annotations:
+                  secrets.stackable.tech/class: kerberos-$NAMESPACE
+                  secrets.stackable.tech/scope: service=access-kafka
+                  secrets.stackable.tech/kerberos.service.names: developer
+              spec:
+                storageClassName: secrets.stackable.tech
+                accessModes:
+                  - ReadWriteOnce
+                resources:
+                  requests:
+                    storage: "1"
+        - name: tls-ca-cert-mount
+          ephemeral:
+            volumeClaimTemplate:
+              metadata:
+                annotations:
+                  secrets.stackable.tech/class: tls
+                  secrets.stackable.tech/scope: pod
+              spec:
+                accessModes:
+                - ReadWriteOnce
+                resources:
+                  requests:
+                    storage: "1"
+                storageClassName: secrets.stackable.tech
+                volumeMode: Filesystem
+      securityContext:
+        fsGroup: 1000
+        runAsGroup: 1000
+        runAsUser: 1000
+      restartPolicy: OnFailure
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: access-kafka-script
+data:
+  script.sh: |
+    set -euxo pipefail
+
+    export KCAT_CONFIG=/stackable/kcat.conf
+    TOPIC=test-topic
+    CONSUMER_GROUP=test-consumer-group
+    BROKER=test-kafka-broker-default-bootstrap.$NAMESPACE.svc.cluster.local:9095
```

Comment:
The discovery ConfigMap `test-kafka` contains `test-kafka-broker-default-bootstrap.kuttl.svc.cluster.local:9093`. It should contain the correct service and port (in this case 9095) and it should be used in the tests.

Reply:
Should be fixed in 109ad3c (tested with kerberos and non-kerberos cases).

Reply:
Test now uses discovery: [6ced9a5](https://github.com/stackabletech/kafka-operator/pull/762/commits/6ced9a5cacdb84ede039144cb66719bfd7574f7a)